pkg/trace/traceutil/truncate_test.go

```diff
@@ -6,17 +6,41 @@
 package traceutil
 
 import (
+	"strings"
 	"testing"
+	"unicode/utf8"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestTruncateString(t *testing.T) {
 	assert.Equal(t, "", TruncateUTF8("", 5))
-	assert.Equal(t, "télé", TruncateUTF8("télé", 5))
+	assert.Equal(t, "tél", TruncateUTF8("télé", 5))
 	assert.Equal(t, "t", TruncateUTF8("télé", 2))
 	assert.Equal(t, "éé", TruncateUTF8("ééééé", 5))
 	assert.Equal(t, "ééééé", TruncateUTF8("ééééé", 18))
 	assert.Equal(t, "ééééé", TruncateUTF8("ééééé", 10))
 	assert.Equal(t, "ééé", TruncateUTF8("ééééé", 6))
+	assert.Equal(t, "", TruncateUTF8("肋", 2))
+}
+
+func FuzzTruncateString(f *testing.F) {
+	f.Add("télé", 5)
```

Comment:
Adding an example using a complex multibyte character leads to more interesting inputs added to the corpus:

```suggestion
	f.Add("télé", 5)
	f.Add("肋", 2)
```

With the first only it plateaus after a few seconds, but adding the second example allows it to keep trying inputs:

`fuzz: elapsed: 13m0s, execs: 65491735 (92419/sec), new interesting: 18 (total: 51)`